src/ZipLoader.js

```diff
@@ -71,6 +71,15 @@ const ZipLoader = class ZipLoader {
 
 			};
 
+			xhr.onerror = ( e ) => {
```

Comment:
`event` is preferable rather than `e` here
```suggestion
			xhr.onerror = ( event ) => {
```